tutorials/editor/upgrading_to_godot_4.rst

```diff
@@ -0,0 +1,540 @@
+.. _doc_upgrading_to_godot_4:
+
+Upgrading to Godot 4
+====================
+
+TODO: https://github.com/godotengine/godot-proposals/issues/387#issuecomment-1251909261
```

Comment:
It's a bit weird as the first thing in the document.

If this is meant to link to further information that can be useful when doing manual porting work, I would link it as a `seealso` further down, where the user would typically need to dig deeper.

Reply:
That was a leftover I didn't intend to keep :slightly_smiling_face: Fixed.